### ISSUE TITLE
Pagination not working: ActionController::ParameterMissing

### ISSUE THREAD

**Issue by Spone** [wearefine/fae#319]
Hello,

I followed the [documentation](https://www.faecms.com/documentation/features-filtering#pagination) to implement pagination on some of my admin controllers.

The pagination is displayed after my index table, but the XHR request fails with the following error:

```
ActionController::ParameterMissing (param is missing or the value is empty: contact):
actionpack (5.1.3) lib/action_controller/metal/strong_parameters.rb:437:in `require'
/usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/controllers/fae/base_controller.rb:93:in `item_params'
/usr/local/opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/fae-fbf5cdb90b78/app/controllers/fae/base_controller.rb:35:in `create'
actionpack (5.1.3) lib/action_controller/metal/basic_implicit_render.rb:4:in `send_action'
actionpack (5.1.3) lib/abstract_controller/base.rb:186:in `process_action'
actionpack (5.1.3) lib/action_controller/metal/rendering.rb:30:in `process_action'
actionpack (5.1.3) lib/abstract_controller/callbacks.rb:20:in `block in process_action'
activesupport (5.1.3) lib/active_support/callbacks.rb:108:in `block in run_callbacks'
[...]
```

The only param passed in the XHR request is the `page`, but `item_params` ([here](https://github.com/wearefine/fae/blob/master/app/controllers/fae/base_controller.rb#L93)) requires a param with the model name.

Is it a bug? Am I using the feature in a wrong way?

Rails 5.1.3, Ruby 2.4.1

Thanks

**Comment by Spone** [comment]
Actually, it's weird that the request performed is a `POST /admin/contacts`, it should actually be a GET in order for it to be processed by the [`index` action](https://github.com/wearefine/fae/blob/master/app/controllers/fae/base_controller.rb#L11).

**Comment by jasonfine** [comment]
It looks like the pagination setup relies on `== fae_filter_form` being present in the index file. I tried this locally and hit the same issue you did, until I added an empty `== fae_filter_form` as the first child of `main.content` (as per the instructions to set up filtering above the pagination instructions)

Perhaps this can be a workaround for you now, and we can look into the possibility of allowing pagination to work without a fae filter form present, or update the docs to state that this is necessary for pagination to work.

Thanks

**Comment by Spone** [comment]
Thanks, it's a workaround. I don't need the search so I used:

```ruby
== fae_filter_form title: '', search: false
```

**Comment by jamesmk** [comment]
fixed in #326

**Comment by lizrush** [comment]
Hi all, I just wanted to share the work around for pagination that I ended up using in place of this one for any one else who might come across this issue. 

I ran into the same issue as @Spone when attempting to implement pagination. Adding the `== fae_filter_form title: '', search: false` does add in that param that was missing, but it also relies on a POST call to the filter action. When I went down this path I ended up polluting my controller since I have a special use case with some subclassing of items that would require me to overwrite the filter action. I was also just not really into the POST call to the filter action.

Since the docs indicate that you can just use kaminari directly to handle pagination and I just wanted to have a simple param based GET request to my index action, I figured that would be the way to go. The problem with this (as you can tell by the related PR #326) the `_filtering.js` code will pick up on the default kaminari class (`.paginate`) and attempt to make the POST call.  

My solution was to find the path of least resistance here. I decided to give the pagination nav element a different class name.

Here’s what I put in my view:
```
  nav.pagination-unfiltered
    == paginate @items, theme: 'fae'
```
(Based on the [fae_paginate method](https://github.com/wearefine/fae/blob/597c70de51395848b7d93f36e2a915dd0471f60d/app/helpers/fae/view_helper.rb#L139))

and then in `fae.scss`:
```
.pagination-unfiltered {
  @extend .pagination
}
```

This is give you the same kaminari styling & theme as provided out of the box, but since you are using the `.pagination-unfiltered` class, no click handlers will be attached to the pagination links. This allows you to use a simple pagination that passes in a page param in the GET call.
